### DISCUSSION TITLE
[bug] Hide Blocks is not selectable

### DISCUSSION THREAD

**Discussion by holema** [YetiForceCompany/YetiForceCRM#16599] [How to] [answered]
# 🐞 bug report

### ⁉️ Describe the bug
Hello, 
in the old version of Yetiforce (4.3) it was possible to hide blocks depending of the role of the user.
No I upgraded to 6.3 and realized that the menu item is not visible anymore. Is this function completly removed or is it somewhere else?

Best regards

**Comment by rskrzypczak** [comment, marked as answer]
Yes, the block hiding panel has been permanently removed.
Familiarize yourself with the functionality of dependent fields.
`Software configuration/Standard modules/Dependent Fields`
![image](https://user-images.githubusercontent.com/10221999/196339520-04fc553b-ebaa-4eb0-9d02-5cf2d2a4b6b6.png)
